src/login/LoginMain/LoginMain.js

```diff
@@ -45,6 +47,11 @@ class Main extends Component {
                   path="/"
                   render={props => <div {...props} ></div>}
                 />
+                <Route
+                  exact
+                  path="/reset/reset-password/"
```

Comment:
I would suggest that we use `path="/reset-password/"` as the base for all of the reset password paths. Let me know what you think.

So, maybe something like this (?):  
1. Give email to get reset password link `path="/reset-password/get-reset-email"`
2. Link has been sent to provided email `path="/reset-password/reset-email-sent"`
3. Email 

**If phone num:**
4. Get a confirmation code to hold on to your identity `path="/reset-password/get-confirmation-code"`
5.  Use the confirmation code `path="/reset-password/use-confirmation-code"`
6 (or no phone num). Choose a new password `path="/reset-password/new-password"`

Reply:
I initially had /reset-password/ as path, but it meant that in the backend the pattern for unauthn views was not standard, and Johan did prefer to use the current /reset/reset-password/.